### PR TITLE
Calendar now rendering

### PR DESCRIPTION
**Pull request by eryoon**

### CODE REVIEW

**Comment by mirman-mtaggart** [comment]
No `for` loops. Use `map`, please.

**Comment by eryoon** [comment]
@mirman-mtaggart Wait a minute... how am I supposed to use `map`? Map is for arrays, but all I have is the _size_ of the array.

**Comment by mirman-mtaggart** [comment]
I see. In that case, make sure your props are getting passed as integers. Right now, they are strings.

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import {render} from "react-dom";
+//import my components
+import Calendar from "./react_components/Calendar"
 
 render(
-    <h1>Hello world!</h1>, document.getElementById("root")
+    <Calendar days="4" periods="8"/>, document.getElementById("root")
```
Comment (mirman-mtaggart) [inline src/index.js]: These props should be ints. `days={4}`, for example.

File: src/react_components/Calendar.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Table, Label} from 'semantic-ui-react';
+export default class Calendar extends React.Component {
+  render() {
+    //header
+    const days = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"];
```
Comment (mirman-mtaggart) [inline src/react_components/Calendar.js]: This feels clunky. Maybe extract the letter days to a separate `config` file?

**Comment by eryoon** [comment]
Closing. Fixed suggestions in PR #2

**Pull request closed without merging**